main.go

```diff
@@ -69,10 +70,27 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		opsFailed.Inc()
 		return
 	}
+
+	whitelist := []string{"http://localhost", "https://searchspring.github.io/snapp-explorer"}
```

Comment:
This list should be read in from an environment variable and always ensuring that the localhost:3827 is always added.